static/js/components/tiles/map_tile.tsx

```diff
@@ -204,13 +215,25 @@ export const fetchData = async (
     .get(`/api/place/parent/${place.dcid}`)
     .then((resp) => resp.data);
   try {
-    const [geoJson, placeStat, population, parentPlaces] = await Promise.all([
-      geoJsonPromise,
-      placeStatPromise,
-      populationPromise,
-      parentPlacesPromise,
-    ]);
-    const rawData = { geoJson, placeStat, population, parentPlaces };
+    const [geoJson, placeStat, population, parentPlaces, borderGeoJsonData] =
+      await Promise.all([
+        geoJsonPromise,
+        placeStatPromise,
+        populationPromise,
+        parentPlacesPromise,
+        borderGeoJsonPromise,
+      ]);
+    // Only draw borders for containing places without 'wall to wall' coverage
+    const shouldDrawBorders =
+      NO_FULL_COVERAGE_PLACE_TYPES.includes(enclosedPlaceType);
+    const borderGeoJson = shouldDrawBorders ? borderGeoJsonData : undefined;
```

Comment:
nit: don't need to make shouldDrawBorders its own constant, could just do the `.includes` check here

Reply:
Switched to using the `shouldShowBorder` function you talked about in your last comment for dryness